src/fontra/views/editor/scene-controller.js

```diff
@@ -504,6 +504,41 @@ export class SceneController {
     return undoInfo !== undefined;
   }
 
+  async doDelete() {
+    await this.editInstance((sendIncrementalChange, instance) => {
+      const { point: pointSelection } = parseSelection(this.selection);
```

Comment:
We'll also need to handle components (`instance.component`). This will be easier.

I think we should move the point logic to a new function in path-functions.js: `deleteSelectedPoints(path, pointIndices)` (everything inside the recordChanges block).

And implement the component logic here. Iterate over the componentIndices in reverse.

Reply:
I also think that the doDelete implementation can stay in editor.js, like breakContour and friends.